core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheConfig.java

```diff
@@ -33,9 +33,9 @@ public class ChunkCacheConfig extends AbstractConfig {
             + "where \"-1\" represents infinite retention";
     private static final long DEFAULT_CACHE_RETENTION_MS = 600_000;
 
-    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetching.size";
+    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetch.max.size";
```

Comment:
nit: could we align the constant name?
```suggestion
    private static final String CACHE_PREFETCH_MAX_SIZE_CONFIG = "prefetch.max.size";
```

Reply:
Fixed